src/components/property-tree/property-tree.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useEffect, useState } from "react";
+// @ts-ignore
+import SortableTree from "react-sortable-tree";
+
+import { ValidChildAsset } from "../../utils/test-fixtures";
+
+import { Asset } from "@mtfh/common/lib/api/asset/v1/types";
+import "react-sortable-tree/style.css";
+
+interface PropertyTreeProps {
+  asset: Asset;
+  childAssets: Asset[] | ValidChildAsset[] | undefined;
+}
+
+interface AssetWithParentsAndChildren {
```

Comment:
isn't this a bit misleading? it might not have either of those. Could we call it TreeAsset or something like that?

Reply:
Sure can, will change that